spec/struct.dd

```diff
@@ -434,6 +452,12 @@ assert(u.b == [2, 0]);
 
 $(H2 $(LNAME2 anonymous, Anonymous Structs and Unions))
 
+    $(P An anonymous struct or union can be declared as a member of a
+    parent class, struct or union by omitting an identifier.
+    Such a declaration simultaneously declares an anonymous type and a
+    hidden field of that type in the parent type. The fields are
```

Comment:
It doesn't really declare a type, it just affects the `.offsetof` of the members. 'Hidden field' usually refers to context pointers or vtables, but that's not what's going on here. This description makes it appear more complex than it is, I would say that an anonymous `struct` and `union` specifies whether the contained member variables are being laid out sequentially or overlapped.

Reply:
Thanks, reworded.